history/characters/460000_drust.txt

```diff
@@ -0,0 +1,24 @@
+# dynasty=70200
+460000={ # High Thornspeaker of the Drust
```

Comment:
He's not on Kul Tiras in 583.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/80979199-192e5100-8e38-11ea-9d4e-3e616f5fb7a0.png)

</details>

Reply:
og holders got replaced by a patron character, fixed